recipes_source/distributed_device_mesh.rst

```diff
@@ -0,0 +1,153 @@
+Getting Started with DeviceMesh
+=====================================================
+
+**Author**: `Iris Zhang <https://github.com/wz337>`__, `Wanchao Liang <https://github.com/wanchaol>`__
+
+.. note::
+   |edit| View and edit this tutorial in `github <https://github.com/pytorch/tutorials/blob/main/recipes_source/distributed_device_mesh.rst>`__.
+
+Prerequisites:
+
+- `Distributed Communication Package - torch.distributed <https://pytorch.org/docs/stable/distributed.html>`__
+- Python
+- PyTorch 2.2
+
+
+Setting up the NVIDIA Collective Communication Library (NCCL) communicators for distributed communication during distributed training can pose a significant challenge. For workloads where users need to compose different parallelisms,
```

Comment:
```suggestion
Setting up distributed communicators, i.e. NVIDIA Collective Communication Library (NCCL) communicators for distributed training can pose a significant challenge. For workloads where users need to compose different parallelisms,
```

Reply:
Since DeviceMesh does not only support NCCL (i.e. user can register custom backend too), let's be clear on this